client/header/activity-panel/activity-header/style.scss

```diff
@@ -27,3 +27,24 @@
 		background: $core-grey-light-700;
 	}
 }
+
+.woocommerce-layout__inbox-panel-header {
+	padding: 15px 43px 10px;
+	h3 {
+		margin: 5px 0;
+		font-weight: 500;
+	}
+	span {
+		margin-left: 7px;
+		background: #40464d;
+		border-radius: 0.8em;
```

Comment:
Do we have a `border-radius` mixin? I noticed that you used a mixin further along.

Reply:
I tried to find a `border-radius` mixin in our code but I couldn't find it. Do you think that it's necessary to start using them when we want to use a `border-radius`?